recipes/websocketpp/all/conanfile.py

```diff
@@ -34,18 +34,20 @@ def source(self):
         self._patch_sources()
 
     def _patch_sources(self):
-        for patch in self.conan_data["patches"][self.version]:
-            tools.patch(**patch)
+        if "patches" in self.conan_data and self.version in self.conan_data["patches"]:
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
 
     def package(self):
         self._patch_sources()
         self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
-        # We have to copy the headers manually, since the current cmake.install() step in the 0.8.1 release doesn't do so.
+        # We have to copy the headers manually, since the upstream cmake.install() step doesn't do so.
         self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + "/websocketpp")
 
     def package_info(self):
         if self.options.asio == "standalone":
             self.cpp_info.defines.extend(["ASIO_STANDALONE", "_WEBSOCKETPP_CPP11_STL_"])
 
-    def package_id(self):
+    def package_id(self):	
         self.info.header_only()
```

Comment:
Do these too declarations conflict? should the options/defines the package id?

Reply:
there is a trailing space, please, remove it.